server/service/src/sync/api_v6/mod.rs

```diff
@@ -0,0 +1,113 @@
+mod core;
+
+use repository::RepositoryError;
+use reqwest::Url;
+use serde::{Deserialize, Serialize};
+use thiserror::Error;
+use util::format_error;
+
+use crate::i64_to_u64;
+
+pub use self::core::*;
+
+use super::{
+    api::{
+        CommonSyncRecord, ParsedError, ParsingResponseError, SyncApiError, SyncApiErrorVariantV5,
+    },
+    translations::PushSyncRecord,
+    SyncApiSettings,
+};
+
+#[derive(Deserialize, Debug, Error, Serialize)]
```

Comment:
Most types here are used in the API and are shared between omSupply central and remote, type safety should be guaranteed for the same version, but need to think about this when version changes

Reply:
Yep, I think keeping separate API version for backward compatibility of all sites would have way too many use cases to keep track of, I think we would take similar approach to what is being done for legacy mSupply central sync api, try to keep it backwards compatible, if we do break compatibility it would require all sites to update to new version. I think this would be possible if we 'sync' binary to remote sites, something that has been mentioned, maybe for RnD day